nncf/experimental/post_training/statistics/statistic_point.py

```diff
@@ -0,0 +1,45 @@
+from collections import UserDict
+
+from nncf.common.tensor_statistics.collectors import TensorStatisticCollectorBase
+from nncf.common.tensor import TensorType
+
+from nncf.experimental.onnx.graph.transformations.commands import ONNXTargetPoint
+
+
+class StatisticPoint:
```

Comment:
Please, add a short description for these classes.

Reply:
Done